teams/3/teams.css

```diff
@@ -0,0 +1,594 @@
+    justify-content: space-between;
+    /* align-items: center; */
+    padding: 24px 24px 28px;
+    background: #ffffff;
+    /* Gray/200 */
+
+    border: 1px solid var(--color-gray-200);
+    box-sizing: border-box;
+    border-radius: 16px;
+    width: 768px;
+    margin-bottom: 24px;
+    height: 168px;
+}
+
+@media (max-width: 1200px) {
+    .open-position {
+        width: 100%;
+        margin-top: 20px;
+    }
+}
+
+.open-position h4 {
+    margin: 0;
+    color: var(--color-gray-900);
+}
+
+.open-position p {
+    margin: 0;
+    color: var(--color-gray-500);
+}
+
+.work-box p {
+    margin-right: 20px;
+}
+
+@media (max-width: 768px) {
+    .open-position {
+        flex-direction: column;
+        height: 100%;
+    }
+    .open-position p {
+        margin-top: 16px;
+        margin-bottom: 32px;
+    }
+}
+
+@media (max-width: 480px) {
+    .open-position h4 {
+        margin-bottom: 8px;
+    }
+    .open-position .header {
+        order: 1;
+        flex-direction: column;
+    }
+    .open-position .country {
+        order: 2;
+    }
+    .open-position p {
+        order: 3;
+    }
+    .salary-range {
+        order: 4;
+    }
+}
+
+.position-info {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.open-position img {
+    margin-right: 6px;
+}
+
+.picture {
+    display: flex;
+    justify-content: center;
+    padding-top: 2rem;
+    padding-left: 5rem;
+    padding-right: 5rem;
+    padding-bottom: 5rem;
+    position: relative;
+    background: url('../../img/office.png');
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 400px;
+}
+
+@media (max-width: 768px) {
+    .picture {
+        padding-top: 2rem;
+        padding-left: 1rem;
+        padding-right: 1rem;
+        padding-bottom: 5rem;
+        background: url('../../img/office-square.png');
+        background-repeat: no-repeat;
+        background-position: center;
+    }
+}
+
+@media (max-width: 480px) {
+    .picture img {
+        width: 343px;
+        height: 240px;
+        object-fit: cover;
+    }
+}
+
+.join {
+    height: 426px;
+}
+
+.inner-box {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    height: 330px;
+    background-color: var(--color-primary-50);
+}
+
+.inner-box h2 {
+    color: var(--color-primary-900);
+    margin-bottom: 0;
+}
+
+.inner-box span {
+    color: var(--color-primary-700);
+}
+
+.inner-box input {
+    box-sizing: border-box;
+    width: 360px;
+    height: 48px;
+    padding: 12px 16px;
+}
+
+@media (max-width: 480px) {
+    .join {
+        display: none;
+    }
+}
+
+.subscribe-btn {
+    background-color: var(--color-purple-600);
+    width: 118px;
+    height: 48px;
+    color: #fff;
+    border: 1px solid #7f56d9;
+    box-sizing: border-box;
+    box-shadow: 0px 1px 2px rgba(16, 24, 40, 0.05);
+    border-radius: 8px;
+    margin-left: 16px;
+}
+
+footer {
```

Comment:
replace all the elements styles with proper class styles, considering all the agreements mentioned at class